shared/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -151,7 +151,7 @@ namespace datadog::shared::nativeloader
             {
                 if (process_name == exclude_assembly)
                 {
-                    Log::Info("CorProfiler::Initialize ClrProfiler disabled: ", process_name," found in default exclude list");
+                    Log::Info("CorProfiler::Initialize Datadog SDK disabled: ", process_name," found in default exclude list");
```

Comment:
Unrelated, but why are they called `default_exclude_assemblies` and `exclude_assembly` when it's a list of process names? 😅

Reply:
Did a PR to fix that (and other minor annoyances) here: https://github.com/DataDog/dd-trace-dotnet/pull/7475